### ISSUE TITLE
Not really much of an issue

### ISSUE THREAD

**Issue by mak448a** [mishalhossin/TicTacToe-QT5#1]
This isn't really too much of an issue with your project, but just letting you know that this is already there by default on QT Designer:

![image](https://github.com/mishalhossin/TicTacToe-QT5/assets/94062293/4ba45ea9-52b1-479f-a437-4387922fc9eb)

**Comment by mishalhossin** [comment]
I didnt even know qt designer existed lmfao really cool

**Comment by mak448a** [comment]
Yeah, I use QT designer mostly, and only manually add UI (very rarely) if it needs to be dynamic. You can see it in action [here](https://github.com/mak448a/QTCord).
